demos/smart_classroom_demo/main.cpp

```diff
@@ -1047,6 +1016,8 @@ int main(int argc, char* argv[]) {
 
                 const int start_frame = 0;
                 const int end_frame = face_obj_id_to_action_maps.size();
+                const int smooth_window_size = static_cast<int>(cap->fps() * FLAGS_d_ad);
```

Comment:
Same here - probably need to do something special for non-video inputs.

Reply:
I don't think we can do something special and useful for non-video inputs